### DISCUSSION TITLE
Jenkins Plugin for Dependency Track

### DISCUSSION THREAD

**Discussion by nihal1993** [DependencyTrack/dependency-track#1309] [Q&A]
Hi, I am trying to integrate Dependency Track using Jenkins Plugin. I have provided URL and API Key. But it is saying connection failed. I am trying to use both the applications in EC2 instances and connect it from there. I have used Dependency Track release 5.1.2 and 4.0.0. I have also tried with .hpi file of version 3.0.2. Could you please tell me how to connect both?

**Comment by stevespringett** [comment]
I'm assuming you've followed the instructions here: https://plugins.jenkins.io/dependency-track/

Also, have you verified that the Dependency-Track URL configured in the Jenkins plugin is the URL to the API Server and not the frontend?
